### DISCUSSION TITLE
After splitting a Seurat object, how to save a RDS file for each sub-object?

### DISCUSSION THREAD

**Discussion by S-W-Hsu** [satijalab/seurat#7433] [General]
Hi,

After I use SplitObject function to split a Seurat object into sub-objects by "ident", how should I save each sub-object to RDS files separately? Thank you.

**Comment by ChristinaSteyn** [comment]
You can probably just use a for loop to iterate through the object and save each part of the object individually. E.g something like this:

for (i in 1:length(split_seurat)){
name <- levels(Idents(split_seurat[[i]]))
saveRDS(split_seurat[[i]], paste0("directory/", name, "_seu.rds"))
}
